odl/solvers/nonsmooth/primal_dual_hybrid_gradient.py

```diff
@@ -18,13 +18,13 @@
 from odl.operator import Operator
 
 
-__all__ = ('pdhg',)
+__all__ = ('pdhg', 'pdhg_stepsize')
 
 
 # TODO: add dual gap as convergence measure
 # TODO: diagonal preconditioning
 
-def pdhg(x, f, g, L, tau, sigma, niter, **kwargs):
+def pdhg(x, f, g, L, niter, tau, sigma, **kwargs):
```

Comment:
If you do this API change, please also switch the roles of `f` and `g`, see #1197

Reply:
They should not be switched here, should they? All other (nonsmooth )solvers have this kind of signature:
```python
def douglas_rachford_pd(x, f, g, L, tau, sigma, niter, callback=None, **kwargs):
def admm_linearized(x, f, g, L, tau, sigma, niter, **kwargs):
def forward_backward_pd(x, f, g, L, h, tau, sigma, niter, callback=None, **kwargs):
```